src/main/java/dp/xlsx/DatasetFormatter.java

```diff
@@ -197,19 +219,40 @@ private void addMetadata() {
         // title row
         Row row = sheet.createRow(rowOffset);
         Cell cell = row.createCell(columnOffset);
-        cell.setCellStyle(headingRightAlignStyle);
+        cell.setCellStyle(workBookStyles.getHeaderRightAlignStyle());
         final String titleLabel = "Title";
         cell.setCellValue(titleLabel);
 
         dimensionColumnWidths.put(columnOffset, titleLabel.length());
 
         cell = row.createCell(columnOffset + 1);
-        cell.setCellStyle(headingStyle);
+        cell.setCellStyle(workBookStyles.getHeadingStyle());
         cell.setCellValue(datasetMetadata.getTitle());
         rowOffset++;
 
         // Add a blank row at the bottom of the metadata.
         sheet.createRow(rowOffset);
         rowOffset++;
     }
+
+
+    private void addNote(final String value) {
+        rowOffset++;
+        Row row = sheet.createRow(rowOffset);
+        row.createCell(0); //Blank cell
+        Cell cell = row.createCell(1);
+        cell.setCellValue(value);
+        cell.setCellStyle(workBookStyles.getNoteStyle());
+    }
+
+    private void AddUserNotes(UserNotes notes) {
```

Comment:
Again picky can you make this method name start with lower case? This aint golang bro :-p

Reply:
Renamed the method 😞